Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"Black\">\r\n    <option name=\"sdkName\" value=\"Python 3.9 (CSC2033_Team37_23_24)\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.10 (CSC2033_Team37_23_24) (2)\" project-jdk-type=\"Python SDK\" />\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision a982daaa764b98bc2213cb61ee92fbfc5371a376)
+++ b/.idea/misc.xml	(date 1715601129198)
@@ -3,5 +3,5 @@
   <component name="Black">
     <option name="sdkName" value="Python 3.9 (CSC2033_Team37_23_24)" />
   </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.10 (CSC2033_Team37_23_24) (2)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (CSC2033_Team37_23_24)" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
